<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PostDataExportDialog</class>
 <widget class="QDialog" name="PostDataExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Calculation Result</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="formatLabel">
     <property name="text">
      <string>Format:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="formatComboBox">
       <item>
        <property name="text">
         <string>VTK files (ASCII)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VTK files (Binary)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSV files</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Topography Files (*.tpo)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ESRI Shapefiles</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="folderLabel">
     <property name="text">
      <string>Output folder:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="folderEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="folderRefButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="prefixLabel">
     <property name="text">
      <string>Prefix:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="prefixEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="fileLabel">
     <property name="text">
      <string>Output file:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="FilenameEditWidget" name="fileEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="timeRangeGroupBox">
     <property name="title">
      <string>Time</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="dataAllCheckBox">
        <property name="text">
         <string>All timesteps</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="dataStartLabel">
          <property name="text">
           <string>Start:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="SliderWithValue" name="dataStartSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dataEndLabel">
          <property name="text">
           <string>End:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="SliderWithValue" name="dataEndSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="samplingLabel">
          <property name="text">
           <string>Skip rate :</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QSpinBox" name="samplingSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QGroupBox" name="dataRangeGroupBox">
     <property name="title">
      <string>Region</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="fullRegionCheckBox">
        <property name="text">
         <string>Full region</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="iminLabel">
        <property name="text">
         <string>I Min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="SliderWithValue" name="iminSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="imaxLabel">
        <property name="text">
         <string>I Max</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="SliderWithValue" name="imaxSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="jminLabel">
        <property name="text">
         <string>J Min</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="SliderWithValue" name="jminSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="jmaxLabel">
        <property name="text">
         <string>J Max</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="SliderWithValue" name="jmaxSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="kminLabel">
        <property name="text">
         <string>K Min</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="SliderWithValue" name="kminSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="kmaxLabel">
        <property name="text">
         <string>K Max</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="SliderWithValue" name="kmaxSlider">
        <property name="maximum">
         <number>40</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>9</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="detailButton">
       <property name="text">
        <string>Show &amp;Detail</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SliderWithValue</class>
   <extends>QSlider</extends>
   <header>guibase/widget/sliderwithvalue.h</header>
  </customwidget>
  <customwidget>
   <class>FilenameEditWidget</class>
   <extends>QLineEdit</extends>
   <header>guibase/widget/filenameeditwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PostDataExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PostDataExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
